lib/fiware-pep-steelskin.js

```diff
@@ -74,14 +74,22 @@ function handleError(error, req, res, next) {
  * @param {Function} next        Call to the next error handler in the chain.
  */
 function accountInfoError(error, req, res, next) {
-    accessLogger.info('Wrong Attempt' +
+    var accessMsg = 'Wrong Attempt';
+    accessMsg = proxyMiddleware.checkAccessMatches(req, accessMsg);
+    accessLogger.info(accessMsg +
                       ' | Error=' + error.name +
                       ' | Token=' + req.headers['x-auth-token'] +
-                      ' | Origin=' + req.connection.remoteAddress +
+                      ' | Origin=' + (req.ip || req.connection.remoteAddress) +
                       ' | UserId=' + req.userId +
+                      ' | ServiceId=' + req.serviceId +
+                      ' | UserName=' + req.userName +
                       ' | Service=' + req.service +
+                      ' | SubServiceId=' + req.subserviceId +
                       ' | SubService=' + req.subService +
                       ' | Action=' + req.action +
+                      ' | Path=' + req.path +
+                      ' | Query=' + JSON.stringify(req.query) +
```

Comment:
En el fichero `lib/middleware/proxy.js` se trunca el body a 100 caracteres (` ' | Body=' + JSON.stringify(req.body).slice(0, 100) + // not all body`), y aquí no, ¿cual es el motivo?

Reply:
Hablado con Álvaro, por lo visto se loga el body entero cuando hay error, y solo una parte cuando no lo hay,

Reply:
El body se trunca a 100 caracteres en caso de acceso valido y se deja entero en caso de error

Reply:
Faltaba documentarlo: https://github.com/telefonicaid/fiware-pep-steelskin/pull/545